site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,293 @@
++++
+date = "2018-09-10T12:21:53-05:00"
+title = "Citra Progress Report - 2018 Q2"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Hello everyone to the summer (or winter, for those on the southern half of this
+Earth) edition of Citra's progess reports. Although not very many changes have
+happened, most have been focused on accuracy improvements and sorely needed user
+experience improvements.
+
+Before we go further into this, our [patreon](https://www.patreon.com/citraemu)
+poll results are out.
+For those of you who didn't know, we conducted a poll on our patreon page,
+exclusively for our patrons, asking them which feature they would like us
+to work on next. Here are the results and as you will see, some of these
+have been considered and already implemented.
+
+
+To get access to such exclusives in the future and to support this project,
+consider becoming a patron.
+Now, enough appetizing, have your entr&eacute;e:
+
+## [Add pause, speed limit hotkeys](https://github.com/citra-emu/citra/pull/3594) by [valentinvanelslande](https://github.com/valentinvanelslande)
+
+Let's start with a simple but sweet one &emdash; you can now assign hotkeys to
+pausing, unpausing, and toggling speed limiting. By default, you can pause with
+<kbd>F4</kbd> and toggle speed limiting with <kbd><kbd>ctrl</kbd>+<kbd>Z</kbd></kbd>.
+But, of course, you can bind these actions to other keys in Citra's `qt-config.ini`.
+The property for pausing is `Shortcuts\Main%20Window\Continue\Pause\KeySeq` and
+for speed toggling it is `Shortcuts\Main%20Window\Toggle%20Speed%20Limit\KeySeq`.
+
+<!-- Hidden as this isn't merged yet.
+## [Make hotkeys configurable via GUI](https://github.com/citra-emu/citra/pull/3786) by [Adityarup Laha](https://github.com/adityaruplaha)
+
+Although we have had hotkeys for a while now, not being able to configure them
+to our needs was a bummer. With this, you can now configure and customize
+hotkeys via the configuration window. However, we don't support binding hotkeys
+to controllers as of now. -->
```

Comment:
The PR isn't merged yet, but if you are going to add it, you should probably add an image of the configuration widget